test/algorithms/disjoint/disjoint_point_box_geometry.cpp

```diff
@@ -95,6 +94,29 @@ void test_all()
         "BOX(1941 2066, 2055 2166)", false);
 }
 
+template <typename P>
+void test_open()
+{
+    using box = bg::model::box<P>;
+    using ring_cw = bg::model::ring<P, true, false>;
+    //using polygon_cw = bg::model::polygon<P, true, false>;
+    //using mpolygon_cw = bg::model::multi_polygon<polygon_cw>;
+    //using ring_ccw = bg::model::ring<P, false, false>;
+    using polygon_ccw = bg::model::polygon<P, false, false>;
+    using mpolygon_ccw = bg::model::multi_polygon<polygon_ccw>;
+    
+    // https://github.com/boostorg/geometry/issues/837
```

Comment:
I would quote issue `#XXX` number in commit message for easier lookup and metadata-integrity 😊

Reply:
Done. :)